napari/components/experimental/chunk/_config.py

```diff
@@ -18,17 +18,25 @@
 
 # NAPARI_ASYNC=1 will use these default settings:
 DEFAULT_ASYNC_CONFIG = {
```

Comment:
Weren't these changes already in async-8? See https://github.com/napari/napari/pull/1684/files#diff-19a4eb1e1e6ca1d1e9a5d5905af8e06bR20-R21. Any idea why they appear again here in this diff?

Reply:
I guess I hadn't merged in master with `async-8` yet. I merged it now. The lines went away.